vm/src/stdlib/math.rs

```diff
@@ -231,6 +233,20 @@ fn math_gcd(a: PyIntRef, b: PyIntRef, vm: &VirtualMachine) -> PyResult {
     Ok(vm.new_int(a.as_bigint().gcd(b.as_bigint())))
 }
 
+fn math_factorial(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(value, None)]);
+    let value = objint::get_value(value);
+    match value {
```

Comment:
Maybe instead of a `match` statement, you could choose the `if` statement like this:
```rust
if value <BigInt::zero() {
    ...
} else if value <= BigInt::one() {
    ...
} else {
    ...
}

Reply:
I used `if` statement before but as far as I know rust could compile with zero-cost abstration so, I think, there is no performance issue and I just feel `match` statement looks much more rustic(?).
That's why I use `match` statement.
Is there more specific reason for using `if` statement..?

Reply:
Oh, but `if` statement is much simpler than `match` in this case. I'll change it right away. thx.

Reply:
`match` is indeed very powerful, but in some cases, a simple `if` does the job.